building.sk

```diff
@@ -752,6 +752,7 @@ caves and cliffs update part 1:
 	deepslate lapis [lazuli] ore¦s = minecraft:deepslate_lapis_ore
 	deepslate diamond ore¦s = minecraft:deepslate_diamond_ore
 	
-	[any] stone ore = any ore
+	[any] stone ore = gold ore, iron ore, coal ore, lapis ore, diamond ore, redstone ore, emerald ore, copper ore
```

Comment:
Instead of having to specify all of the ores again, you can just use the `any ore` alias - it also helps out with speeding up loading times for aliases :)
Like this:
```suggestion
	[any] stone ore = any ore, copper ore
```